multicluster/controllers/multicluster/memberclusterannounce_controller.go

```diff
@@ -140,6 +143,35 @@ func (r *MemberClusterAnnounceReconciler) Reconcile(ctx context.Context, req ctr
 		// Nothing to handle in this case and MemberClusterAnnounce will also be deleted soon.
 		// TODO: Add ClusterClaim webhook to make sure it cannot be deleted while ClusterSet is present.
 	}
+
```

Comment:
We can move mapLock.Unlock() here? @luolanzone

Reply:
I will double check this mapLock today.

Reply:
Moved the comment before the data block